waffle-e2e/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@ethereum-waffle/e2e",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "ts-node buildTestContracts",
+    "test:nobuild": "export NODE_ENV=test && mocha",
+    "test": "yarn build && yarn test:nobuild",
+    "lint": "eslint '{src,test}/**/*.ts'",
+    "lint:fix": "eslint --fix '{src,test}/**/*.ts'"
+  },
+  "dependencies": {
+    "@ethereum-waffle/compiler": "^3.0.0-beta.1"
```

Comment:
This project should probably only have dev dependencies

Reply:
I left it here because of linter warning in buildTestContracts

Reply:
add a linter comment in the script then to disable the error

Reply:
The linter is configured so that only tests can use dependencies from devDependencies